src/components/patient/patients/patientContainer.tsx

```diff
@@ -34,12 +36,56 @@ const PatientContainer: React.FC<PatientContainerProps> = props => {
       patients.filter(patient => tableUtils.filter(patient, search, ['_id', 'maritalStatus', 'createdAt', 'createdBy']))
     );
 
+  const patientsInDebtOrCreditMenu = (
+    <Menu>
+      <Menu.Item
+        key='1'
```

Comment:
keys should be like the value not random numbers

Reply:
like incredit/indebt?

Reply:
Yes